### ISSUE TITLE
Bug : doesn't work with webps

### ISSUE THREAD

**Issue by nathanaelhoun** [calibreapp/image-actions#31]
I don't know how to configurate this action to recognize my webps images (it works well on all the others images).
I have this output in github action :
```
- Processing: /github/workspace/docs/images/background.webp
[Error: Input file contains unsupported image format]
```
Any clue ?

**Comment by benschwarz** [comment]
Hey @nathanaelhoun, I'm not sure! Do you have a public repo that highlights this behaviour? I'd love to do some testing from our side to see what's up.

**Comment by nathanaelhoun** [comment]
Hello @benschwarz, I've just changed [this one (personal webpage)](https://github.com/nathanaelhoun/nathanaelhoun.fr) to public for you. You can see in the last action job log that it didn't worked ;-)

**Comment by henrikemota** [comment]
I have this same problem with my images

**Comment by benschwarz** [comment]
Thanks for the verification @henrikemota. We'll get to this when we can

**Comment by henrikemota** [comment]
Hello, have any updates on this bug?

**Comment by chenrui333** [comment]
bump this issue again, any ETA?

**Comment by henrikemota** [comment]
Hello @benschwarz, have any updates on this bug?

**Comment by nathanaelhoun** [comment]
Hi @benschwarz, is there any update on this?

**Comment by benschwarz** [comment]
Closed by #47
